Dockerfile

```diff
@@ -1,12 +1,10 @@
-FROM ruby:3.2.0-alpine3.17@sha256:c690d3b45ef65e0098104c272f1736b0595b824908d5639c3a3d17636581a905  AS build
+FROM ruby:3.2.0-alpine3.17@sha256:c690d3b45ef65e0098104c272f1736b0595b824908d5639c3a3d17636581a905 AS build
 
 WORKDIR /app
 
-# Edge repo is necessary for Node 16 and openssl 3
-RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories
 RUN apk update
 RUN apk upgrade --available
-RUN apk add libc6-compat openssl-dev build-base libpq-dev nodejs=18.15.0-r0 npm
```

Comment:
Is there a reason for removing the specific version of nodejs?

Reply:
Thats interesting because we have 18.14.1 in the `.nvmrc` file.

We know that in PaaS the buildpack didn't include that version which is why we ended up with 18.15.0 here.


If you leave in the node version  then it seems to want to use node 19 (i.e latest node version rather than LTS version)

Reply:
I though using 19 that was a side effect of `edge`, no? It was installing 18.15.0 fine before

Reply:
@cadmiumcat `18-14.2-r0` might be coming straight from the alpine version https://pkgs.alpinelinux.org/packages?name=nodejs&branch=v3.17&repo=&arch=&maintainer=

Reply:
Are we happy with that? [Last time we had decided to go to 18.15.0](https://github.com/alphagov/forms-admin/commit/2c9292839a171bffbe788e2f669687e3a5fdfc81)

Reply:
This is a good point. I think the correct solution would be to build a specific node version in the container. That is more work and a bit of a hassle though.

It looks like [alpine 3.17 is tracking node 18](https://gitlab.alpinelinux.org/alpine/aports/-/commit/680c2e279b7e926bfe84e7b90e818c7bb11f33fb) and looks like it would stick to node even numbers for LTS.

Could we change our .nvm to just `v18`? Node is only included for building assets and isn't included in the image which runs so we would be able to pickup issues quickly, I think.